examples/va/nfv/sriov/control-plane.md

```diff
@@ -0,0 +1,70 @@
+# Configuring networking and deploy the OpenStack control plane
+
+## Assumptions
+
+- A storage class called `local-storage` should already exist.
+
+## Initialize
+
+Switch to the "openstack" namespace
+```
+oc project openstack
+```
+Change to the nfv/sriov directory
+```
+cd architecture/examples/va/nfv/sriov
+```
+Edit the [network-values.yaml](network-values.yaml) and
+[service-values.yaml](service-values.yaml) files to suit 
+your environment.
+```
+vi network-values.yaml
+vi service-values.yaml
+```
+Alternatively use your own copyies of those files and edit 
+[kustomization.yaml](kustomization.yaml) to use those copies.
+```
+resources:
+  - network-values-ci-framework.yaml
+  - service-values-ci-framework.yaml
+```
+
+Generate the control-plane and networking CRs.
+```
+kustomize build > control-plane.yaml
+```
+
+## Create CRs
+```
+oc apply -f control-plane.yaml
+```
+
+Wait for NNCPs to be available
+```
+oc wait nncp -l osp/nncm-config-type=standard --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured --timeout=300s
+```
+
+Wait for control plane to be available
+```
+oc wait osctlplane controlplane --for condition=Ready --timeout=600s
+```
+
+## Workaround
```

Comment:
Perhaps the common directory should have a markdown file describing individual workarounds and both this readme and the HCI read me could link to it. That would avoid content duplication.

Reply:
> Perhaps the common directory should have a markdown file describing individual workarounds and both this readme and the HCI read me could link to it. That would avoid content duplication.

Or in `docs` but I don't want to hold this review up over that.

Reply:
Yes, let's clean that up in a follow-up